src/Components/ErrorPopUp/style.module.css

```diff
@@ -0,0 +1,53 @@
+.container{
+  width: 500px;
```

Comment:
@brendobrendo I see you updated the fonts to be `rems` which is great, 
but the `widths, padding and margin` should be percentages (or some other flexible value). 
we want to use as few "hard pixels" as possible for mobile-first design.

sorry I know I kinda keep harping on this, but it saves a lot of styling headaches/tweaks in the future

Reply:
@brendobrendo I see you updated. looks great thanks!